### ISSUE TITLE
App requires two clicks to open from Windows10 tray

### ISSUE THREAD

**Issue by jtsmith0101** [keybase/keybase-issues#3058]
This unintended feature has been annoying me for a bit of time but now I'm writing about it.  

When the app is running but minimized, visible in the system tray, to open the app, I ALWAYS need to perform this workflow, twice.  That is I click on whatever I'm wanting to do at that time (either chat or down arrow "open keybase"), then repeat.  I need to do this workflow twice as the first time will not result in the desired behavior.  Has anyone else complained about this?

**Comment by malgorithms** [comment]
yes, others have complained about this. we hope to have it fixed soon!

cc @zanderz @chrisnojima

**Comment by ThierryIT** [comment]
Same for me on Windows 7

**Comment by d1vanloon** [comment]
Issue remains with Keybase GUI Version: 1.0.36-20171121163724+f68b10e on Windows 10 Pro 1709.
